### DISCUSSION TITLE
Hide app from taskbar

### DISCUSSION THREAD

**Discussion by TzeroOcne** [neutralinojs/neutralinojs#1340] [Q&A] [answered]
Is it possible to hide neutralino app from taskbar? for example in Tauri there is options `skipTaskbar` to hides the window icon from the taskbar and in Qt there is `Qt.Tool` window flag

**Comment by mathiasayivor** [comment]
Hope this is not too late, and 
**Yes, it’s possible** - but Neutralino doesn’t expose a built-in flag for it yet.
You’ll need to do it through an extension by calling the **native OS APIs** directly. On Windows, for example, you can use the Win32 API to update the window’s extended style (`GWL_EXSTYLE`):

* remove `WS_EX_APPWINDOW`
* add `WS_EX_TOOLWINDOW`

This makes the Neutralino window a **tool window**, which removes it from the taskbar (similar to `skipTaskbar` in Tauri or `Qt.Tool`).

**Reply by TzeroOcne** [reply]
Thank you, I was wondering if there's built in way to do it through neutralino, I'm not really into calling win32 api through my neutralino app but oh well, for now I'm just let it be

**Reply by shalithasuranga** [reply, marked as answer]
Hello, please check Neutralinojs's [`skipTaskbar`](https://neutralino.js.org/docs/configuration/neutralino.config.json#modeswindowskiptaskbar-boolean). Thanks :tada:
